hbase_storage/hbase1.0.0.x/src/main/java/com/splicemachine/access/hbase/HPartitionCreator.java

```diff
@@ -55,6 +55,7 @@ public PartitionCreator withName(String name){
 
     @Override
     public PartitionCreator withDisplayNames(String[] displayNames){
+        descriptor.setValue(SIConstants.SCHEMA_DISPLAY_NAME_ATTR, displayNames[0]);
         descriptor.setValue(SIConstants.TABLE_DISPLAY_NAME_ATTR, displayNames[0] != null ? displayNames[0] : descriptor.getNameAsString());
```

Comment:
In the other platforms you forgot to update (increment by 1) the displayNames indexes for the TABLE/INDEX attributes.

Reply:
Oops, will make those changes

Reply:
@dgomezferro Done, Thanks for reviewing.